ui/apps/platform/src/Containers/VulnMgmt/Entity/Policy/VulnMgmtPolicyOverview.js

```diff
@@ -55,6 +63,11 @@ const noop = () => {};
 const VulnMgmtPolicyOverview = ({ data, entityContext, setRefreshTrigger }) => {
     const workflowState = useContext(workflowStateContext);
     const { isFeatureFlagEnabled } = useFeatureFlags();
+    const [page, setPage] = useState(0);
```

Comment:
I'm not sure if it is appropriate to put the pagination and sorting for the child table in `workflowState`, since that seemed to be used primarily for when the workflow sidebar is a full table.

LMK if this really should be handled in the `workflowState` though.